### PR TITLE
Lazy Images: do not include js linting config in production

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* This file does not need to be shipped to folks using the production version of the package.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Not much to test here.

#### Proposed changelog entry for your changes:

* N/A

### CODE REVIEW

File: packages/lazy-images/.gitattributes [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Files not needed to be distributed in the package.
+.eslintrc.js        export-ignore
 .gitattributes      export-ignore
```
Comment (anomiex) [inline packages/lazy-images/.gitattributes:3]: The don't need .gitignore either.
```suggestion
.eslintrc.js        export-ignore
.gitattributes      export-ignore
.gitignore          export-ignore
```

**Review by anomiex — APPROVED** [review]
Seems ok to me. One suggestion for improvement.

**Comment by anomiex** [comment]
Needs updating for the repo reorg. Otherwise seems good.

**Review by kraftbj — DISMISSED** [review]

**Comment by kraftbj** [comment]
It looks good... but is it for the right package?

**Comment by jeherve** [comment]
> is it for the right package?

uh. 🤯 

When I merged the repo reorg changes, the auto-merge didn't handle things very well it seems :)

I changed that in b5c5153f1fce18bd02829e8cb569089cdf74b36f.

**Review by kraftbj — COMMENTED** [review]
Much better!

**Review by kraftbj — APPROVED** [review]

**Pull request merged**